var/spack/repos/builtin/packages/py-tensorflow-hub/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyTensorflowHub(Package):
+    """TensorFlow Hub is a library to foster the publication, discovery, and
+    consumption of reusable parts of machine learning models."""
+
+    homepage = "https://github.com/tensorflow/hub"
+    url = "https://pypi.io/packages/py2.py3/t/tensorflow_hub/tensorflow_hub-0.11.0-py2.py3-none-any.whl"
```

Comment:
Since PyPI only has wheels, let's download and install from GitHub instead: https://github.com/tensorflow/hub/archive/refs/tags/v0.12.0.tar.gz